varcode/variant_collection.py

```diff
@@ -110,49 +67,55 @@ def __str__(self):
     def __repr__(self):
         return str(self)
 
-    def clone(self, new_variants=None):
+    def clone_metadata(self, new_variants):
```

Comment:
Gotcha. I'd be fine with any name for this method, as long as there is only one method